niteshkumarniranjan/embercalculator/app/styles/app.css

```diff
@@ -0,0 +1,115 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font: 400 21px Quicksand, sans-serif;
+}
+
+body {
+  background: #4C4C4B;
+}
+
+#calculator {
+	width: 280px;
+	height: auto;
+  margin: 3em;
+  box-shadow: 0 0 0 1px #9b9b9b;
+  border-radius: 2px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-left: -140px;
+  margin-top: -200px;
+}
+
+.top span.clear {
+	float: left;
+}
+
+.top .screen {
+  height: 130px;
+  width: 100%;
+	float: right;
+	padding: 0 10px;
+	background: #e3e7ea;
+  font-size: 52px;
```

Comment:
Be consistent with your indentations. Don't add unnecessary indentations.

Reply:
Okay!